### PR TITLE
chore(website): Publish Gateway 1.4.0 changelog

### PR DESCRIPTION
**Pull request by jamilbk**
Publishes the 1.4.0 Gateway changelog and fixes a couple other minor issues.

### CODE REVIEW

File: website/src/components/Changelog/GUI.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default function GUI({ title }: { title: string }) {
         <ChangeItem pull="7123">
           Reports the version to the Portal correctly.
         </ChangeItem>
-        <ChangeItem pull="6996">
+        <ChangeItem pull="6996" enable={title === "Windows"}>
```
Comment (jamilbk) [inline website/src/components/Changelog/GUI.tsx]:
```suggestion
        <ChangeItem pull="6996" enable={title === "Linux GUI"}>
```

File: website/src/components/Changelog/GUI.tsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ export default function GUI({ title }: { title: string }) {
         <ChangeItem pull="7123">
           Reports the version to the Portal correctly.
         </ChangeItem>
-        <ChangeItem pull="6996">
+        <ChangeItem pull="6996" enable={title === "Linux GUI"}>
```
Comment (ReactorScram) [inline website/src/components/Changelog/GUI.tsx:26]: Good catch

File: website/src/components/Changelog/Gateway.tsx [hunk 3]
```diff
@@ -10,24 +10,24 @@ export default function Gateway() {
...
-            Disable by setting `FIREZONE_NO_TELEMETRY=1`.
-          </ChangeItem>
-          <ChangeItem pull="7164">
-            Fixes an issue where the Gateway would fail to accept connections and
-            had to be restarted.
-          </ChangeItem>
-      </Unreleased>
+      <Entry version="1.4.0" date={new Date("2024-11-04")}>
+        <ChangeItem pull="6960">
+          Segregates traffic restrictions beween DNS Resources CIDR Resources,
```
Comment (ReactorScram) [inline website/src/components/Changelog/Gateway.tsx]: "Separates" is probably a more common word

**Review by ReactorScram — APPROVED** [review]

File: website/src/components/Changelog/Gateway.tsx (hunk 3 above)
Comment (jamilbk) [reply website/src/components/Changelog/Gateway.tsx]: 👍

Comment (jamilbk) [inline website/src/components/Changelog/Gateway.tsx]:
```suggestion
          Separates traffic restrictions beween DNS Resources CIDR Resources,
```

**Pull request merged**